src/main/java/uk/gov/companieshouse/ordernotification/emailsendmodel/CertifiedCopy.java

```diff
@@ -1,13 +1,23 @@
 package uk.gov.companieshouse.ordernotification.emailsendmodel;
 
+import java.util.List;
+import java.util.Objects;
+
 public class CertifiedCopy {
     private String id;
-    private String dateFiled;
-    private String type;
-    private String description;
     private String companyNumber;
     private String deliveryMethod;
-    private String fee;
+    private List<FilingHistoryDetailsModel> filingHistoryDetailsModelList;
+
+    public CertifiedCopy(String id,
+            String companyNumber,
+            String deliveryMethod,
+            List<FilingHistoryDetailsModel> filingHistoryDetailsModelList) {
+        this.id = id;
+        this.companyNumber = companyNumber;
+        this.deliveryMethod = deliveryMethod;
+        this.filingHistoryDetailsModelList = filingHistoryDetailsModelList;
+    }
```

Comment:
Change to private api sdk java to make a FilingHIstoryDocumentApi a single item.